edtf/natlang/en.py

```diff
@@ -13,19 +15,46 @@
 DEFAULT_DATE_1 = datetime(1234, 1, 1, 0, 0)
 DEFAULT_DATE_2 = datetime(5678, 10, 10, 0, 0)
 
-SHORT_YEAR_RE = r"(-?)([\dX])([\dX])([\dX])([\dX])"
-LONG_YEAR_RE = r"Y(-?)([1-9]\d\d\d\d+)"
-CENTURY_RE = r"(\d{1,2})(c\.?|(st|nd|rd|th) century)\s?(ad|ce|bc|bce)?"
-CE_RE = r"(\d{1,4}) (ad|ce|bc|bce)"
+SHORT_YEAR_RE = re.compile(r"(-?)([\du])([\dxu])([\dxu])([\dxu])")
```

Comment:
This regex is different. I think u was superseded by X in the updated spec: https://www.loc.gov/standards/datetime/

Reply:
I've removed it as it isn't actually used anywhere.